internal/algod/participation/participation.go

```diff
@@ -101,15 +120,18 @@ func RemovePartKeyByID(slice *[]api.ParticipationKey, id string) {
 	}
 }
 
-func FindParticipationIdForVoteKey(slice *[]api.ParticipationKey, votekey []byte) *string {
-	for _, item := range *slice {
+// FindParticipationIdForVoteKey searches a List for an item's VoteParticipationKey matching the given votekey and returns its ID.
+// It returns a pointer to the ID as a string if a match is found, or nil otherwise.
+func FindParticipationIdForVoteKey(slice List, votekey []byte) *string {
+	for _, item := range slice {
 		if string(item.Key.VoteParticipationKey) == string(votekey) {
 			return &item.Id
 		}
 	}
 	return nil
 }
 
+// ToLoraDeepLink generates a Lora deep link URL for a transaction wizard based on network, offline state, and participation key.
```

Comment:
are we using the toLoraDeepLink function anywhere now that we do the redirects?

Reply:
No usages anymore, we can clean it up.